src/plugins/orangehrmOpenidAuthenticationPlugin/Controller/OpenIdConnectLoginController.php

```diff
@@ -44,40 +44,33 @@ class OpenIdConnectLoginController extends AbstractVueController implements Publ
      */
     public function getHomePageService(): HomePageService
     {
-        if (!$this->homePageService instanceof HomePageService) {
-            $this->homePageService = new HomePageService();
-        }
-        return $this->homePageService;
+        return $this->homePageService ??= new HomePageService();
     }
 
     /**
-     * @throws OpenIDConnectClientException
+     * @throws OpenIDConnectClientException|BadRequestException
```

Comment:
incomplete doc comment
```suggestion
     * @param Request $request
     * @return RedirectResponse
     * @throws OpenIDConnectClientException|BadRequestException
```